miniforge3/ubuntu/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM ubuntu:bionic
```

Comment:
Why does it matter that it is ubuntu over debian?

Reply:
The primary reason is that Ubuntu has timed long term support releases (every 2y), while debian does not. When people want to use this base image, but add new packages from apt, it should ideally be as up to date as possible. Longer term, this is more likely with Ubuntu than Debian, primarily because of the release cadence. You can get around this to some extent with using their rolling releases, but that makes reproducibility very hard by default. 

Jupyter's docker-stacks base images switched from debian to ubuntu in https://github.com/jupyter/docker-stacks/pull/428 for these reasons. repo2docker (and hence binder) use ubuntu as base default for these reasons as well. 

Some of this is definitely personal preference of course, so am happy to move to debian if people would like to - although I'd personally prefer to not!

Reply:
[MAC (Mandatory Access Control) > Implementations](https://en.wikipedia.org/wiki/Mandatory_access_control#Implementations):

- Debian: seccomp, *optional* SELinux w/ incomplete policy set
- Ubuntu: AppArmor, seccomp, *optional* SELinux w/ incomplete policy set
- Fedora/CentOS/RHEL: seccomp, SELinux in enforcing mode
- Android (aarch64/x86; termux): seccomp, SELinux in enforcing mode

yum:
```Dockerfile
FROM centos:7
# RUN yum update -y
RUN yum install -y wget bzip2 ca-certificates git && yum clean all && rm -rf /var/cache/yum
```

dnf:

```Dockerfile
FROM centos:8
# FROM fedora:32
# ...
# RUN dnf update -y
RUN dnf install -y wget bzip2 ca-certificates git && dnf clean all && rm -rf /var/cache/yum /var/cache/dnf
```